pkg/arvo/app/ping.hoon

```diff
@@ -345,6 +120,11 @@
     |=  old=state-1
     ^-  state-2
     old(- %2)
+  ::
+  ++  state-2-to-3
+    |=  old=state-2
+    ^-  state-3
+    [%3 %formal %.n (galaxy-for our.bowl bowl)]
```

Comment:
This scry in `+on-load` is a problem. Instead, we should either a) pick out the galaxy (there's exactly one, right?) from the set of ships in `old`, or b) initialize to our prefix galaxy: `(head (flop (^saxo:title our.bowl)))`.